plugins/CoreHome/templates/_dataTable.twig

```diff
@@ -50,6 +57,14 @@
         {% if error is defined %}
             <div vue-entry="CoreHome.Alert" severity="danger">{{ error.message }}</div>
         {% else %}
+            {% if showTableActionsInHeader %}
+                <div class="row dataTableHeaderControls">
```

Comment:
The `Live_spec` tests have some selectors that get tripped by this new element:

https://github.com/matomo-org/matomo/blob/9a3ef94df61682e7d543f3f8b438c368f6eb71a6/plugins/Live/tests/UI/Live_spec.js#L34

This would now need to be something like this:

```
var report = await page.jQuery('.dataTableVizVisitorLog .card.row:eq(1)');
```

And there are probably more selectors in that spec that need to be updated accordingly.

Reply:
This shouldn't trip on this, as the Visits Log does not have a card on the top level, so the first row for the table controls is not within a card.

Reply:
Updated the selectors